play-services-base/src/main/java/org/microg/gms/common/MultiConnectionKeeper.java

```diff
@@ -36,6 +36,7 @@
 import static android.os.Build.VERSION.SDK_INT;
 import static android.os.Build.VERSION_CODES.ICE_CREAM_SANDWICH;
 import static org.microg.gms.common.Constants.GMS_PACKAGE_NAME;
+import static org.microg.gms.common.Constants.MICROG_GMS_PACKAGE_NAME;
```

Comment:
Wouldn't be better something like USER_GMS_PACKAGE_NAME?
Both are still microG.

Reply:
done